src/content/guides/how-to-use-a-wallet/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: How to use a wallet
+description: A guide explaining how to send, receive tokens and connect to web3 projects.
+lang: en
+---
+
+# How to: Use a wallet
+
+Learn how to operate all the basic functions of a wallet. If you don’t have one yet, check out our How to: "Register" an Ethereum account.
+
+
+## Open your wallet
+
+You should see a dashboard that will likely show your balance and contain buttons to Send and Receive tokens. 
+
+## Receive cryptocurrency
+
+Do you want to receive crypto into your wallet?
+
+Each wallet has its own receiving Ethereum address which is a unique sequence of numbers and letters. The address functions like a bank account number. Ethereum address will always start with “0x”.
+
+You need to provide the sender with your receiving address. Many wallet apps let you copy to clipboard the address or show a QR code to scan for easier usage. 
+
+1. Open your wallet app.
+2. Click on "Receive" or similarly worded option on the dashboard
+3. Copy your Ethereum address to clipboard.
+4. Provide the sender with your receiving Ethereum address.
+
+It is the same process if you are trying to send your funds from an exchange.
+
+## Send cryptocurrency
+
+Would you like to send ETH to another wallet? 
+
+1. Prepare the receiving address and the network of the recipient.
+2. Open your wallet app.
+3. Click on a “Send” button on the dashboard of your wallet or similarly worded alternative.  
+4. Enter the receiving address or scan a QR code with your camera so that you don’t have to write the address manually.
+<br/><br/>
+<img src="./send.png" width="400" alt="Send field for crypto address"/>
+<br/><br/>
+
+4. Many assets, like DAI or USDC, exist on multiple networks. When transferring crypto tokens, make sure that the other side is using the same network as you are. 
+5. Confirm the transaction and have prepared a small amount of ETH to pay for a transaction fee, the exact amount of the fee changes depending on network congestion. 
+6. Once your transaction is processed, the crypto will show up in the recipient’s account. This might take anywhere from few seconds to few minutes.
+
+## Connecting to projects
+
+Your address represents you on all Ethereum projects. You will not need to register individually on any project. Once you have a wallet, you can connect to any Ethereum project without any additional information. No emails or any other personal information are needed.
+
+1. Visit any project’s website.
+2. Click a button “Open the App” in the menu if the main page is a presentation of the project. 
+3. Once you are in the app click on “Connect”
+
+<br/><br/>
+<img src="./connect1.png" width="400" alt="Button allowing user to connect to the website with a wallet"/>
+<br/><br/>
```

Comment:
@wackerow if i change this, will i loose the 400px width rule? The image is otherwise too big, should i resize it down first? or can we keep this way of adding it to the page?